lsp-origami.el

```diff
@@ -57,18 +57,28 @@
 	       (lsp-origami--folding-range-to-fold range create))
 	     (lsp--get-nested-folding-ranges))))
 
+;;;###autoload
+(defun lsp-origami-try-enable ()
+  "Turn on `origami-mode' locally and try to enable `lsp-origami-mode'."
+  (interactive)
+  (origami-mode 1)
+  (cond ((lsp--capability "foldingRangeProvider")
+         (lsp-origami-mode 1))
+        ((called-interactively-p 'any)
```

Comment:
can you log using lsp-log when it is not supported and not called interactively?